mscore/startcenter.cpp

```diff
@@ -75,7 +75,7 @@ Startcenter::Startcenter(QWidget* parent)
 
             horizontalLayout->addWidget(_webView);
             
-            //workaround for the crashes sometimes happend in Chromium on macOS with Qt 5.12
+            //workaround for the crashes sometimes happpening in Chromium on macOS with Qt 5.12
```

Comment:
one p too much ;-)

Reply:
yikes, thanks. I'll update to codespell dictionary to catch this. e2ca31e